django_recaptcha/apps.py

```diff
@@ -1,11 +1,11 @@
 from django.apps import AppConfig
 from django.core.checks import Tags, register
 
-from captcha.checks import recaptcha_key_check
+from django_recaptcha.checks import recaptcha_key_check
 
 
-class CaptchaConfig(AppConfig):
-    name = "captcha"
+class DjangoRecaptchaConfig(AppConfig):
```

Comment:
Code itself doesn't need to change. Just the imports

Reply:
Figured I changed it to consistently apply the new naming. Would it hurt to keep it like this?